pkg/client/docker/types.go

```diff
@@ -14,6 +14,7 @@ type TagResponse struct {
 type Result struct {
 	Name      string  `json:"name"`
 	Timestamp string  `json:"last_updated"`
+	Digest    string  `json:"digest,omitempty"`
```

Comment:
not sure this is in the response
https://docs.docker.com/reference/api/hub/latest/#tag/repositories/paths/~1v2~1namespaces~1%7Bnamespace%7D~1repositories~1%7Brepository%7D~1tags/get

Reply:
I discovered it whilst looking up / using some test/sample data - I.E: within the current response:

https://registry.hub.docker.com/v2/repositories/davidcollom/hub3-exporter/tags